src/storage/src/sink/kafka.rs

```diff
@@ -409,6 +409,8 @@ impl TransactionalProducer {
             .key(&self.progress_key);
         self.producer.send(record).map_err(|(e, _)| e)?;
 
+        fail::fail_point!("kafka_sink_commit_transaction");
```

Comment:
Is this change intended?

Reply:
Ah, this is the change by Nikhil.

Reply:
Yes! It adds the fail point, where we sleep for 5 s in the test, which is the way to make the test reliable.